src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -793,6 +815,11 @@ protected void BuildMethods(INamedTypeSymbol type, IndentedStringBuilder b, Plat
 		{
 			foreach (var method in type.GetMembers().OfType<IMethodSymbol>())
 			{
+
+				if (method.ToString().Contains("GetStringForUri"))
+				{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)